content/deploy/community-cloud/get-started/explore-your-workspace.md

```diff
@@ -5,36 +5,53 @@ slug: /deploy/streamlit-community-cloud/get-started/explore-your-workspace
 
 # Explore your workspace
 
-If you just [created your account](/deploy/streamlit-community-cloud/get-started/create-your-account), congrats! You are now logged in and ready to go. If you are joining someone else's workspace you may already see apps populated in your workspace. If not, then you need to deploy an app! Check out our next section on how to [Deploy your app](/deploy/streamlit-community-cloud/deploy-your-app). If you need an app to deploy, check out our <a href="https://streamlit.io/gallery" target="_blank">App gallery</a> which includes apps for machine learning, data science, and business use cases.
+If you just [created your account](/deploy/streamlit-community-cloud/get-started/create-your-account) and [connected your GitHub account](/deploy/streamlit-community-cloud/get-started/connect-your-github-account), congrats! You are now logged in and ready to go. If you are joining someone else's workspace you may already see some apps.
```

Comment:
There's a mix on the site, but I've been moving toward "sign in" for consistency since that's what the UI says. So yes, we can update existing instances to "sign."